package.json

```diff
@@ -87,18 +87,17 @@
     "@visx/shape": "^3.5.0",
     "@visx/tooltip": "^3.3.0",
     "assimpjs": "^0.0.7",
-    "babel-loader": "^8.2.3",
-    "babel-plugin-lodash": "^3.3.4",
+    "babel-loader": "^9.2.1",
     "bottleneck": "^2.19.5",
     "catalog-converter": "^0.0.9",
     "classnames": "^2.3.1",
     "clipboard": "^2.0.8",
-    "commander": "^11.1.0 ",
+    "commander": "^12.1.0 ",
```

Comment:
Commit 0a451d13ea downgraded commander to 11.1.0 so it would work with node 16.

I'm not against bumping the engine version though, I would prefer to bump engines to node >= 20 so we could "easily" get rid of the security issues in terriajs-server as well while we're at it.

Reply:
Agree that we should bump the engine version, will look into it separately. In this case though commander is only used for a couple of CLI things and the CI hasn't tripped on it. So it should be fine i guess.

Reply:
The previous commander problem that caused the downgrade was most likely fixed by commit 88e587c66e0d4.